src/front-end/typescript/lib/pages/opportunity/create.tsx

```diff
@@ -107,10 +104,9 @@ const view: component_.page.View<State, InnerMsg, Route> = () => {
           className="mb-4 mb-md-0"
           description={
             <span>
-              Use a <em>Team With Us</em> opportunity to Lorem ipsum dolor sit
-              amet, consectetur adipiscing elit, sed do eiusmod tempor
-              incididunt ut labore et dolore magna aliqua. Amet risus nullam
-              eget felis eget nunc lobortis mattis.
+              Use a <em>Team With Us</em> opportunity to to procure individual
```

Comment:
Looks like we have an extra "to" here.  Let's delete the extra one.

Reply:
complete